fission/src/systems/physics/PhysicsSystem.ts

```diff
@@ -719,6 +721,45 @@ class PhysicsSystem extends WorldSystem {
         JOLT.destroy(this._joltBodyInterface)
         JOLT.destroy(this._joltInterface)
     }
+
+    /**
+     * Creates a ghost object and a distance constraint that connects it to the given body
+     * The ghost body is part of the LAYER_GHOST which doesn't interact with any other layer
+     * The caller is responsible for cleaning up the ghost body and the constraint
+     *
+     * @param id The id of the body to be attatched to and moved
+     * @returns The ghost body and the constraint
+     */
+
+    public CreateGodModeBody(id: Jolt.BodyID): [Jolt.Body, Jolt.Constraint] {
+        const body = this.GetBody(id)
+        const bodyPosition = body.GetPosition()
+        const ghostPosition = new THREE.Vector3(bodyPosition.GetX(), bodyPosition.GetY(), bodyPosition.GetZ())
+        const ghostBody = this.CreateBox(new THREE.Vector3(0.05, 0.05, 0.05), undefined, ghostPosition, undefined)
+
+        const ghostBodyId = ghostBody.GetID()
+        this._joltBodyInterface.SetObjectLayer(ghostBodyId, LAYER_GHOST)
+        this._joltBodyInterface.AddBody(ghostBodyId, JOLT.EActivation_Activate)
+        this._bodies.push(ghostBodyId)
+
+        const constraintSettings = new JOLT.PointConstraintSettings()
+        const constraint = constraintSettings.Create(ghostBody, body)
```

Comment:
```suggestion
        constraintSettings.set_mPoint1(anchorPoint)
        constraintSettings.set_mPoint2(anchorPoint)
        const constraint = constraintSettings.Create(ghostBody, body)
```
You may want to explicitly state Constraint Space for readability

Reply:
The anchor point is shared because it's a point constraint, right?

Reply:
@Ewie21 These are the mPoint1 and mPoint2 changes